core/templates/pages/about-page/about-page.component.html

```diff
@@ -12,15 +12,15 @@ <h2 class="oppia-about-title oppia-about-section-title" [innerHTML]="'I18N_ABOUT
       <p class="oppia-about-content oppia-about-section-content" [innerHTML]="'I18N_ABOUT_PAGE_SECTION_ONE_CONTENT' | translate"></p>
       <div class="oppia-about-button-container">
         <oppia-primary-button
-          (onClickPrimaryButton)="onClickVisitClassroomButton()"
-          [customClasses]="['btn', 'oppia-about-button', 'oppia-about-button-browse']"
+          [buttonHref]="onClickVisitClassroomButton()"
+          [customClasses]="['oppia-about-button', 'oppia-about-button-browse']"
           buttonText="{{ 'I18N_ACTION_BROWSE_LESSONS' | translate }}">
         </oppia-primary-button>
       </div>
       <div class="oppia-about-button-container">
         <oppia-primary-button
-          (onClickPrimaryButton)="onClickAccessAndroidButton()"
-          [customClasses]="['btn', 'oppia-about-button', 'oppia-about-button-browse']"
+          buttonHref="/android"
```

Comment:
Why is it `[buttonHref]` above and `buttonHref` here?

Reply:
In Angular, `buttonHref` is used when it's a literal static value like "/android" and `[buttonHref]` is used when it's a dynamic variable or a function or similar